### PR TITLE
exclude hdr10+ from DV HDR10 CF

### PR DESCRIPTION
**Pull request by garypiner**
# Pull request

**Purpose**
The "DV HDR10" currently matches with "DV.HDR10" and "DV.HDR10+", but not "DV.HDR10Plus". This is to update the CF to be consistent and only match with "DV.HDR10"

**Approach**
This is the current regex https://regex101.com/r/YQXE5R/1
This is the new regex https://regex101.com/r/KQO62Q/1
It only matches "DV.HDR10"

**Open Questions and Pre-Merge TODOs**
Check all boxes as they are completed

- [x] Use github checklists. When solved, check the box and explain the answer.

**Learning**
If you're adding a new Custom Format make sure you follow the [Radarr/Sonarr Custom Format (JSON) Guidelines](https://github.com/TRaSH-/Guides/blob/master/.github/CONTRIBUTING.md).

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-/Guides/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/json/radarr/cf/dv-hdr10.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(HDR(10)?)\\b)(?=.*\\b(DV|DoVi)\\b)"
+        "value": "^(?=.*\\b(HDR(10)?(?!\\+))\\b)(?=.*\\b(DV|DoVi)\\b)"
```
Comment (bakerboy448) [inline docs/json/radarr/cf/dv-hdr10.json:14]:
```suggestion
        "value": "^(?=.*\\b(HDR(10)?(?!\\+|P(?:lus)?))\\b)(?=.*\\b(DV|DoVi)\\b)"
```
cover Plus too?

File: docs/json/radarr/cf/hdr10plus-boost.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
       "negate": true,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(HDR(10)?)\\b)(?=.*\\b(DV|DoVi)\\b)"
+        "value": "^(?=.*\\b(HDR(10)?(?!\\+))\\b)(?=.*\\b(DV|DoVi)\\b)"
```
Comment (bakerboy448) [inline docs/json/radarr/cf/hdr10plus-boost.json:23]: why is this needed?

"name": "Not HDR10", covers this

File: docs/json/radarr/cf/hdr10.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
       "negate": true,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(HDR(10)?)\\b)(?=.*\\b(DV|DoVi)\\b)"
+        "value": "^(?=.*\\b(HDR(10)?(?!\\+))\\b)(?=.*\\b(DV|DoVi)\\b)"
```
Comment (bakerboy448) [inline docs/json/radarr/cf/hdr10.json:22]: why is this needed?

"name": "Not HDR10+" prevents this

File: docs/json/radarr/cf/dv-hdr10.json (hunk 1 above)
Comment (garypiner) [reply docs/json/radarr/cf/dv-hdr10.json:14]: It doesn't need to be explicitly included as the word boundary prevents it from being included

File: docs/json/radarr/cf/hdr10plus-boost.json (hunk 2 above)
Comment (garypiner) [reply docs/json/radarr/cf/hdr10plus-boost.json:23]: Just did a search and replace for the current DV HDR10 regex and replaced it in all files

File: docs/json/radarr/cf/hdr10.json (hunk 3 above)
Comment (TRaSH-) [reply docs/json/radarr/cf/hdr10.json:22]: we changed the CF a while ago to make it better matching, and to make it easier to update we prefer to keep the conditions consistent.
but we don't want to go fully one liners

**Review by TRaSH- — APPROVED** [review]
The Team took a look and it looks good, appreciate the find and the PR
I only updated the regex example
